packages/manager/modules/freefax/package.json

```diff
@@ -17,11 +17,11 @@
     "ovh-manager-webfont": "^1.2.0"
   },
   "devDependencies": {
-    "@ovh-ux/component-rollup-config": "^7.0.0 || ^8.0.0"
+    "@ovh-ux/component-rollup-config": "^8.0.0 || ^9.0.0"
   },
   "peerDependencies": {
-    "@ovh-ux/manager-ng-layout-helpers": "^1.0.0 || ^2.0.0",
     "@ovh-ux/manager-core": "^11.0.0 || ^12.0.0",
+    "@ovh-ux/manager-ng-layout-helpers": "^1.0.0 || ^2.0.0",
```

Comment:
```suggestion
    "@ovh-ux/manager-ng-layout-helpers": "^2.0.0",
```
Nice catch

Reply:
Changes has been applied by `sort-package-json`.
Maybe we can handle it in a separate Pull Request (see following search results).

```sh
$ git grep "@ovh-ux/manager-ng-layout-helpers" -- "*.json"
packages/manager/apps/dedicated/package.json:    "@ovh-ux/manager-ng-layout-helpers": "^1.0.1 || ^2.0.0",
packages/manager/apps/freefax/package.json:    "@ovh-ux/manager-ng-layout-helpers": "^1.1.0 || ^2.0.0",
packages/manager/apps/hub/package.json:    "@ovh-ux/manager-ng-layout-helpers": "^1.0.0 || ^2.0.0",
packages/manager/apps/overthebox/package.json:    "@ovh-ux/manager-ng-layout-helpers": "^1.1.0 || ^2.0.0",
packages/manager/apps/sms/package.json:    "@ovh-ux/manager-ng-layout-helpers": "^1.1.0 || ^2.0.0",
packages/manager/apps/telecom/package.json:    "@ovh-ux/manager-ng-layout-helpers": "^1.0.0 || ^2.0.0",
packages/manager/modules/freefax/package.json:    "@ovh-ux/manager-ng-layout-helpers": "^1.0.0 || ^2.0.0",
packages/manager/modules/hub/package.json:    "@ovh-ux/manager-ng-layout-helpers": "^1.0.0 || ^2.0.0",
packages/manager/modules/ng-layout-helpers/package.json:  "name": "@ovh-ux/manager-ng-layout-helpers",
packages/manager/modules/ng-layout-helpers/package.json:    "start": "lerna exec --stream --scope='@ovh-ux/manager-ng-layout-helpers' --include-dependencies -- npm run build --if-present",
packages/manager/modules/ng-layout-helpers/package.json:    "start:dev": "lerna exec --stream --scope='@ovh-ux/manager-ng-layout-helpers' --include-dependencies -- npm run dev --if-present",
packages/manager/modules/ng-layout-helpers/package.json:    "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/manager-ng-layout-helpers' --include-dependencies -- npm run dev:watch --if-present"
packages/manager/modules/overthebox/package.json:    "@ovh-ux/manager-ng-layout-helpers": "^1.0.0 || ^2.0.0",
packages/manager/modules/sms/package.json:    "@ovh-ux/manager-ng-layout-helpers": "^1.0.0 || ^2.0.0",
```

Reply:
See https://github.com/ovh/manager/pull/4255